data/json/items/melee/bludgeons.json

```diff
@@ -1171,6 +1171,42 @@
     "price": "40 USD",
     "price_postapoc": "5 USD"
   },
+  {
+    "id": "survivor_polehammer",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "looks_like": "lucern_hammer",
+    "name": { "str": "survivor polehammer" },
+    "description": "A standard hammerhead solidly mounted at the end of a proper spear shaft.  The long leverage provides surprising balance and impact force.  Simple and effective.",
+    "weapon_category": [ "POLEARMS" ],
+    "material": [ "wood", "steel" ],
+    "flags": [ "DURABLE_MELEE", "REACH_ATTACK", "NONCONDUCTIVE", "POLEARM", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "SWEEP" ],
+    "weight": "1500 g",
+    "volume": "1250 ml",
+    "longest_side": "180 cm",
+    "to_hit": { "grip": "weapon", "length": "long", "surface": "any", "balance": "neutral" },
+    "price": "50 USD",
+    "price_postapoc": "10 USD",
+    "qualities": [ [ "HAMMER", 1 ] ],
+    "category": "weapons",
+    "melee_damage": { "bash": 25 }
+  },
+  {
+    "id": "survivor_polehammer_makeshift",
+    "copy-from": "survivor_polehammer",
+    "type": "GENERIC",
+    "name": { "str": "makeshift survivor polehammer" },
+    "description": "A standard hammerhead mounted at the end of a long stick.  It is far from being a proper weapon, but it allows you to bash zombie heads from a safe distance.",
+    "delete": { "flags": [ "DURABLE_MELEE" ] },
+    "weight": "1800 g",
+    "volume": "1650 ml",
+    "to_hit": { "grip": "solid", "length": "long", "surface": "any", "balance": "uneven" },
```

Comment:
Pretty minor nit, surface isn't any, you want to hit square and aligned with the striking face.  You have a 2 square inch or so impact zone at the end of a meter long pole that needs to make contact,  that's not easy.

Reply:
> Pretty minor nit, surface isn't any, you want to hit square and aligned with the striking face. You have a 2 square inch or so impact zone at the end of a meter long pole that needs to make contact, that's not easy.

Well, you're probably right, I just based this on lucerne hammer. And that weapon is a mess. It has surface "any", even if its hammerhead striking area is not much bigger than standard hammer. IRL their shape is ofter even more reliant on good alighment. And lucerne hammer has pierce damage, probably from beak or spearhead, but they benefit from mentioned "any" surface.

Back to topic, giving these polehammers "point" surface will be huge nerf, then they gonna need increase damage to be similar of metal plated quaterstaves or something

Reply:
Make the change to make it correct, then we can use the DPS tests to determine whether to make other changes to damage/etc.

Reply:
Make the change to make it correct, then we can use the DPS tests to determine what changes to make.